deploy/crds/sriovnetwork.openshift.io_v1_sriovnetworknodepolicy_cr.yaml

```diff
@@ -32,3 +32,23 @@ spec:
     rootDevices: ['0000:02:00.0', '0000:02:00.1']
   deviceType: netdevice
   isRdma: true
+
+---
+# Changing mtu for Mellanox IB NICs is out of scope
```

Comment:
Does the annotation mean when IB linkType is specified, mtu should not be configured to a non-default value?
Why is it out of scope?

Reply:
The MTU is configured by the SM. A user can put a value which is less or equal to the MTU configured in the SM if he want.

Reply:
@moshe010 Do you mean the SM will configure the MTU of the VFs/PFs automatically, so the operator doesn't need to set it?